src/include/txn.h

```diff
@@ -244,8 +244,8 @@ struct __wt_txn {
 
     /*
      * Snapshot data:
+     *	ids >= snap_max are invisible,
```

Comment:
Do we document this anywhere else other than source code? I believe we have a architecture guide page for snapshot as well.

Reply:
WT-6907 is in progressing. I left a comment there for alison to be aware of this change.